app/admin/requests/Menu.php

```diff
@@ -13,7 +13,7 @@ public function rules()
             ['menu_description', 'admin::lang.label_description', 'between:2,1028'],
             ['menu_price', 'admin::lang.menus.label_price', 'required|numeric|min:0'],
             ['categories.*', 'admin::lang.menus.label_category', 'sometimes|required|integer'],
-            ['locations.*', 'admin::lang.column_location', 'sometimes|integer'],
+            ['locations.*', 'admin::lang.column_location', 'sometimes|required|integer'],
```

Comment:
You can remove the `sometimes` here

Reply:
Should we also have locations validated here?